### PR TITLE
Docs: Add Contribution section

### LINKED ISSUE #474: Improve roadmap visibility
Currently, the roadmap can be found by accessing https://github.com/jquery/globalize/milestones and by clicking the "[read] more" link. Although, it's impossible (or at least very difficult) for someone to find it and it requires constant manual updates/maintenance.

We should find out a better solution: (a) to improve the access to this document and (b) to require less manual maintenance.

### PR DESCRIPTION
**Pull request by rxaviers**
Fixes #474

### CODE REVIEW

**Comment by rxaviers** [comment]
@jquery/globalize, @kborchers, what do you think? More description at #474.

**Comment by rxaviers** [comment]
It's missing to list the Globalize related projects. For example, cldr-data, the integration projects (compiler, webpack plugin, react-globalize and its compiler and its webpack plugin) and ask contributors to include others.

File: README.md [hunk 1]
```diff
@@ -638,6 +640,53 @@ handle dependencies and CLDR loading manually yourself.
   [Read more...](doc/error/e-unsupported.md)
 
 
+## Contribution
```
Comment (jzaefferer) [inline README.md]: How about "Contributing"?

File: README.md [hunk 2]
```diff
@@ -638,6 +640,53 @@ handle dependencies and CLDR loading manually yourself.
...
+[currency module]: https://github.com/jquery/globalize/labels/currency%20module
+[date module]: https://github.com/jquery/globalize/labels/date%20module
+[message module]: https://github.com/jquery/globalize/labels/message%20module
+[number module]: https://github.com/jquery/globalize/labels/number%20module
+[plural module]: https://github.com/jquery/globalize/labels/plural%20module
+[relative time module]: https://github.com/jquery/globalize/labels/relative%20time%20module
+
+### Roadmap
+
+To avoid manual maintenance we avoid duplicating information elsewhere, so our
```
Comment (jzaefferer) [inline README.md]: I'd drop the disclaimer, just explain what it is, about the "avoid duplicating" part.

Comment (rxaviers) [inline README.md]: Please, how would you rephrase this paragraph?

File: README.md [hunk 3]
```diff
@@ -638,6 +640,53 @@ handle dependencies and CLDR loading manually yourself.
...
+roadmap is actually the collection of all open issues and pull requests. Having
+said that, we believe it's important to facilitate that the community knows the
+following:
+
+- [Ongoing work][] lists our current sprint. Here you find what we're actively
+  working on at this very moment. Priority is determined by the community needs.
+  If there's anything you want done, feel free to weigh in on a new or existing
```
Comment (Arkni) [inline README.md]: I think this should be: `... you want to be done, ...`

File: README.md [hunk 4]
```diff
@@ -638,6 +640,53 @@ handle dependencies and CLDR loading manually yourself.
...
+  issue or pull request. Pro activity is your best bet.
+- [Next work][] lists what we're going to work next or what we dream of.
```
Comment (Arkni) [inline README.md]: What about using:
```
lists what we're going or planned to work on next.
```

Comment (Arkni) [inline README.md]: What about:
```md
[Next tasks][]
```
instead of
```md
[Next work][]
```

File: README.md (hunk 1 above)
Comment (rxaviers) [inline README.md]: Works for me.

File: README.md (hunk 4 above)
Comment (rxaviers) [inline README.md]: I'm considering to change this to "[Help wanted][] lists potential next work you could help us do now" something in this line.

File: README.md [hunk 5]
```diff
@@ -638,6 +640,53 @@ handle dependencies and CLDR loading manually yourself.
...
+
+[Ongoing work]: https://github.com/jquery/globalize/labels/Current%20Sprint
+[Next work]: https://github.com/jquery/globalize/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22new+feature%22+-label%3A%22Current+Sprint%22+
```
Comment (rxaviers) [inline README.md]: This should be updated to actually list Help Wanted label.

File: README.md (hunk 1 above)
Comment (rxaviers) [inline README.md]: Actually, "Contribution" is more parallel than "Contributing" given the subsequent section "Development".

Comment (rxaviers) [inline README.md]: Rephrased

File: README.md (hunk 3 above)
Comment (rxaviers) [inline README.md]: :+1: ed0a5da

File: README.md (hunk 2 above)
Comment (rxaviers) [inline README.md]: How does 6fe1183 sound?

Comment (Arkni) [inline README.md]: How about something like this ?

```md
### Roadmap

Our roadmap consist of a collection of all opened issues and pull requests.
Having said that, the community should know how to differentiate/separate
between the **Ongoing tasks** and the **Next planned ones**:
```

File: README.md [hunk 6]
```diff
@@ -638,6 +640,51 @@ handle dependencies and CLDR loading manually yourself.
...
+### Roadmap
+
+Our roadmap is the collection of all open issues and pull requests where you can
+find:
+
+- [Ongoing work][] lists our current sprint. Here you find what we're actively
+  working on at this very moment. Priority is determined by the community needs.
+  If there's anything you want to be done, feel free to weigh in on a new or
+  existing issue or pull request. Pro activity is your best bet.
+- [Help wanted][] lists potential next work you could help us do now.
```
Comment (Arkni) [inline README.md]: Something like this sound good: `lists potential next work that you could help us to accomplish`

File: README.md [hunk 7]
```diff
@@ -638,6 +640,51 @@ handle dependencies and CLDR loading manually yourself.
...
+
+### Roadmap
+
+Our roadmap is the collection of all open issues and pull requests where you can
+find:
+
+- [Ongoing work][] lists our current sprint. Here you find what we're actively
+  working on at this very moment. Priority is determined by the community needs.
+  If there's anything you want to be done, feel free to weigh in on a new or
+  existing issue or pull request. Pro activity is your best bet.
```
Comment (scottgonzalez) [inline README.md]: "Pro activity" -> "Being proactive"

**Comment by scottgonzalez** [comment]
The contribution section definitely needs a link to contributing.md.

File: README.md (hunk 7 above)
Comment (rxaviers) [inline README.md]: 20b1462 thx

File: README.md (hunk 6 above)
Comment (rxaviers) [inline README.md]: 87070b9 thx

**Comment by rxaviers** [comment]
> The contribution section definitely needs a link to contributing.md.

Absolutely. Included by 5a72177.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-Welcome! Thanks for your interest in contributing to Globalize. You're **almost** in the right place. More information on how to contribute to this and all other jQuery Foundation projects is over at [contribute.jquery.org](http://contribute.jquery.org). You'll definitely want to take a look at the articles on contributing [code](http://contribute.jquery.org/code).
+Welcome! Thanks for your interest in contributing to Globalize. You're **almost** in the right place. More information on how to contribute to this and all other jQuery Foundation projects is over at [contribute.jquery.org](http://contribute.jquery.org). To help you get started and before you jump into writing code, be sure to read [Writing Code for jQuery Foundation Projects](http://contribute.jquery.org/code/).
```
Comment (rxaviers) [inline CONTRIBUTING.md]: With this change, I wanted to emphasize one should get started with "Writing Code for jQuery Foundation Projects".

Comment (Arkni) [inline CONTRIBUTING.md]: How about something like this:
>Before getting involved and writing code for this project, be sure to read [...]'s section first.

File: README.md (hunk 1 above)
Comment (jzaefferer) [inline README.md]: Or you drop the "Development" header and make that part of "Contributing"

File: CONTRIBUTING.md (hunk 8 above)
Comment (rxaviers) [inline CONTRIBUTING.md]: Simpler :+1: changed by d07f68c

File: README.md (hunk 1 above)
Comment (rxaviers) [inline README.md]: Leaving it as is for now, but we could definitely reevaluate and change it later.

Comment (rxaviers) [inline README.md]: Re-read docs and I see "Getting started". So, I am changing "Contribution" to "Contributing" as you suggest.

Comment (rxaviers) [inline README.md]: 8522383

File: README.md (hunk 4 above)
Comment (rxaviers) [inline README.md]: I go back on my last comment... Naming the last item "Help wanted" has a bad side-effect: it makes the first item sound as not needing help. The reality is: help is always welcome and needed. I am consider to name both items "Ongoing work" and "Everything else".

File: README.md (hunk 5 above)
Comment (rxaviers) [inline README.md]: See https://github.com/jquery/globalize/pull/528#discussion_r42825702

**Pull request closed without merging**